packages/pnpm/package.json

```diff
@@ -16,14 +16,13 @@
     "bin"
   ],
   "optionalDependencies": {
-    "graceful-fs": "4.2.4",
```

Comment:
And `graceful‑fs` is being removed from `optionalDependencies` because ???

Reply:
graceful-fs was removed from all dependencies in the repository (as one of the changes in pnpm v6). Not sure why it was still in pnpm CLI.